client/components/token-field/index.jsx

```diff
@@ -476,6 +508,18 @@ var TokenField = React.createClass( {
 		}
 	},
 
+	_valueContainsToken( token ) {
+		let search = find( this.props.value, ( item ) => {
```

Comment:
>I'm not sure what will make it easier to understand.

Maybe changing the argument name to something that better reflects that it's the string variant to search? `value`, `search`, or `tokenString` maybe?